docs/_assets/js/toolbar.js

```diff
@@ -1,20 +1,26 @@
-$(document).ready(function() {
-  $("#menu-icon").click(() => {
-    $(".sidebar").toggleClass("toggled");
-  })
-  $("#search-icon").click(() => {
-    $("#searchbar").toggleClass("shown");
-    $("#search-api-input").focus();
-  })
-  const searchInput = $("#search-api-input");
-  searchInput.keydown(evt => {
-    if (evt.which == 13) {
-      const baseUrl = $("#baseurl-input").val();
-      window.location = (
-        baseUrl + "/api/search.html?" +
-        "searchTerm=" + searchInput.val() +
-        "&previousUrl=" + encodeURI(window.location)
-      );
+$(function() {
+  const menuIcon = $("#menu-icon");
+  const sidebar = $(".sidebar");
+  menuIcon.on("click", function() {
```

Comment:
The syntax changes to `function() {` here (and in other functions in this file) are a bit weird considering the opposite changes from `(args) => rhs` made in `api-search`. I personally prefer the `(args) => rhs` (since it's more similar to scala). Maybe let's keep the `function() {}` syntax for top level functions, and use lambda syntax for inner functions (like in `api-search`)?

Reply:
Oh! Your right, I meant to make everything Lambda, I had my code prettification settings incorrectly for that file, sorry for the mistake. I'll edit and squash later today.